core/crypto/crypto.cpp

```diff
@@ -185,19 +185,19 @@ void Crypto::_bind_methods() {
 Ref<Resource> ResourceFormatLoaderCrypto::load(const String &p_path, const String &p_original_path, Error *r_error, bool p_use_sub_threads, float *r_progress, CacheMode p_cache_mode) {
 	String el = p_path.get_extension().to_lower();
 	if (el == "crt") {
-		X509Certificate *cert = X509Certificate::create();
+		X509Certificate *cert = X509Certificate::create(true);
 		if (cert) {
 			cert->load(p_path);
 		}
 		return cert;
 	} else if (el == "key") {
-		CryptoKey *key = CryptoKey::create();
+		CryptoKey *key = CryptoKey::create(true);
```

Comment:
Then this change wouldn't be needed.

This applies to all the static `create()` methods that are called manually in some places.